pkg/datapath/loader/base.go

```diff
@@ -361,6 +361,23 @@ func (l *Loader) Reinitialize(ctx context.Context, o datapath.BaseProgramOwner,
 		}
 	}
 
+	if !option.Config.TunnelingEnabled() {
+		if option.Config.EnableIPv4EgressGateway || option.Config.EnableHighScaleIPcache {
```

Comment:
Please use `option.Config.EgressGatewayCommonEnabled()`

Reply:
Thanks! `option.Config.EnableIPv4EgressGateway` is read in a bunch of places throughout the codebase, though. Better patch it up if that's not supposed to happen, or folks will copy-paste.

Reply:
Ah, looks like you changed this line in particular last week, and it probably got dropped in conflict resolution during rebase.